UniversityApp/Controllers/StudentsController.cs

```diff
@@ -105,6 +106,11 @@ public async Task<IActionResult> GetGrades(int id)
         {
             var grades = await _studentsService.GetStudentGrades(id);
 
+            if(grades == null)
```

Comment:
There shouldn't be returned BadRequest() .

If student does not exist then BadRequest is proper one as input data are not correct
but if there is no grades added then we should get 201 status with empty list